### DISCUSSION TITLE
Scatter Map Color Coding By Values, > Sign Does Not Display Correctly

### DISCUSSION THREAD

**Discussion by tinoptilogic** [RevealBi/Reveal.Sdk#120] [General]
When using the Scatter Map in Analytics, and you set Conditional Formatting for the color-coding in the widget settings we see an instance of mojibake, where the > symbol is being replaced by â%.¥

![image](https://github.com/RevealBi/Reveal.Sdk/assets/150904348/9e1a481f-c6d3-46c5-ab77-9cc00d07190d)

**Comment by TorreyBetts** [comment]
Does the HEAD of your html page contain the following?

`<meta charset="UTF-8">`

**Reply by tinoptilogic** [reply]
Yes, it contains the following.

**Reply by TorreyBetts** [reply]
This morning we tested on a few machines and samples and wasn't able to reproduce. Are you able to try and create an isolated sample to test on your end to see if it still occurs outside of your application?
